examples/simple-server.rs

```diff
@@ -151,7 +153,16 @@ async fn init_consensus(
         })
         .collect::<Vec<_>>();
 
+    // Pick a random port for the builder
+    let builder_port = pick_unused_port().expect("failed to get unused port");
+    let builder_url = surf_disco::Url::parse(&format!("127.0.0.1:{builder_port}"))
```

Comment:
Usually we use 0.0.0.0 for a binding address. I recall there was some OS-specific behavior with 127.0.0.1. Don't really remember what the specific issue was, though, so if this works I guess it's fine